databroker/experimental/config.yml

```diff
@@ -0,0 +1,8 @@
+authentication:
+  allow_anonymous_access: true
+trees:
+  - path: /
+    tree: "server_ext:MongoAdapter.from_uri" # "server_ext:MongoAdapter.from_uri"
+    args:
+      uri: mongodb://localhost:27017/fxi-recon-tiled-dev-5
```

Comment:
I wonder, instead of hard-coding example database here, we should use `MongoAdapter.from_mongomock` so that it constructs a temporary database and does not require MongoDB to be running unsecured on `localhost`.